f/assignGear/f_assignGear_3IFB_v.sqf

```diff
@@ -7,7 +7,7 @@
 		clearMagazineCargoGlobal _unit;
 		clearItemCargoGlobal _unit;
 		clearBackpackCargoGlobal _unit;
-		_unit addWeaponCargoGlobal [_carbine, 2];
+		_unit addWeaponWithAttachmentsCargoGlobal [[_carbine,"","","",[_carbinemag,500],[],""],2];
```

Comment:
I'm guessing 500 is a general value to ensure it's always a full mag?

Reply:
Yes, the command automatically trims large numbers to the maximum possible in the given magazine.